content/blog/developer-experience.md

```diff
@@ -0,0 +1,81 @@
+---
+date: 2024-03-15
+title: 'Serverless Workflows: an Automated Developer Experience'
+---
+Great job on installing the Orchestrator plugin and the SonataFlow operator! But what comes next?
+
+If you aim to understand the full development lifecycle of serverless workflows, from zero to production, then you've come to the right place.
+
+Thanks to the Orchestrator functions and automations, developers can now focus solely on building their applications without being burdened by unnecessary cognitive load. Let's delve into how to effectively manage the end-to-end software development lifecycle of serverless workflows,
+leveraging these built-in capabilities. 
+
+## Software Development with Git
+Let's assume your company follows the `feature branches git workflow`:
+* Developers work on individual `feature` branches.
+* The `develop` branch serves as the integration point where all features are merged to validate the application in the staging environment.
+* Once the software receives the green light, the code is released to the `main` branch and deployed to the production environment.
+
+![feature branches git workflow](/blog/images/git-workflow.png)
+
+Don't be surprised, but the Orchestrator plugin automatically installs all the needed resources to handle these steps for you.
+
+## A Reference Architecture for Automated Deployments of Serverless Workflows
+The proposed reference architecture consists of the following components:
+* `Orchestrator Helm chart`: the installer of RHDH Orchestrator .
+* `RHDH`: the Red Hat product for Backstage.
```

Comment:
I'd like to suggest moving this item to the first in the list and to define the acronym as "Red Hat Developer Hub (RHDH)" to increase the readability

Reply:
done but not fully convinced. Please take a look